README.rst

```diff
@@ -29,9 +29,10 @@ common case:
 Client side Requirements
 ------------------------
 
-- sudo, or logged in as root on your client machine.
+- sudo, or root access on your client machine.
   (The server doesn't need admin access.)
 - Python 2.7 or Python 3.5.
+- For Python 2.7, you need PyXAPI, available `here <http://www.pps.univ-paris-diderot.fr/~ylg/PyXAPI/>`_
```

Comment:
I suppose the best place for this could be in the table in the tproxy Requirements section.